README.md

```diff
@@ -62,7 +62,7 @@ And add this line to your welcome view:
 
      <%= link_to "Sign in with Facebook", user_facebook_omniauth_authorize_path %>
 
-If you visit your page and click "Sign in with Facebook," you should be able to go through a Facebook application authorization flow.
+If you visit your page(`users/sign_in`) and click "Sign in with Facebook," you should be able to go through a Facebook application authorization flow.
```

Comment:
Y'all are both right! Thats what happens when you close 15 pull requests on a repo in one shot 😊